### PR TITLE
Feat: Added Alert Component

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
An Alert component stating different situations is added.

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    background-color: var(--success-color-lt);
+}
+
+ion-icon {
+    font-size: large;
+}
+
+ion-icon:hover {
+    opacity: 0.5;
+}
```
Comment (hinanshisuthar) [inline Components/Alert/alert.css:50]: Add a new line at the EOF

**Review by hinanshisuthar — COMMENTED** [review]
Everything else looks perfectly fine. Good to merge.

File: Components/Alert/alert.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Components/Alert/alert.css:50]: For some reason, even if i add a new line at EOF, VSCode doesn't consider the space.
Do you have any solution for this? If yes, can you please share?

**Review by ZuberDunge — COMMENTED** [review]
Looks good to me

**Review by SukanyaSen530 — APPROVED** [review]

**Pull request merged**